client/my-sites/themes/helpers.js

```diff
@@ -139,8 +139,12 @@ var ThemesHelpers = {
 		let basePath = route.sectionify( path );
 		let analyticsPageTitle = 'Themes';
 
+		if ( tier ) {
+			basePath += '/type/:tier';
```

Comment:
I think this was missing for `/design` tracking, wasn't it? (for `pageEvent.record()` again())